src/coverlet.core/coverlet.core.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyVersion>5.1.1</AssemblyVersion>
     <IsPackable>false</IsPackable>
+    <LangVersion>preview</LangVersion>
```

Comment:
Should master use preview version of C#?

Reply:
Good question!
I did it for test project https://github.com/tonerdo/coverlet/blob/master/test/coverlet.core.tests/coverlet.core.tests.csproj
At the moment the only feature I used is null coalesce `&&=` because other features need new netcoreapp3.0 object.
Also corefx seem allow preview https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L248
@ViktorHofer do you see any issue with this config related to emitted IL?

Reply:
No concerns from my side.